panther_on_sonoma/.env.cpu

```diff
@@ -0,0 +1,3 @@
+DISPLAY=${DISPLAY:?You need to define display env}
```

Comment:
The `DISPLAY` env is doubled in gpu config. Maybe we should move this env directly to the compose file?

Reply:
I would leave it as it is, all the information needed about displaying the gpu and cpu is encapsulated. However I changed name to inform that this is display configuration.